presentations/installation/1.sh

```diff
@@ -7,11 +7,8 @@ $ rustup default stable
 # Override the default toolchain in your directory
 $ rustup override set stable
 
-# Display documentation in browser
```

Comment:
Any reason we remove this? I typically mention this during trainings.

Reply:
oh okay! I wouldn't mention it *there* because from what I've seen so far you'd usually build the docs using cargo, not rustup directly, and that is covered in more depth later